src/whitenoise/users.rs

```diff
@@ -316,8 +427,59 @@ impl User {
                 e
             })?;
 
-        self.sync_relay_urls(whitenoise, relay_type, &network_relay_urls)
-            .await
+        match network_relay_result {
+            Some((network_relay_urls, event_timestamp, event_id)) => {
+                // Convert Nostr timestamp to DateTime<Utc> (convert seconds to milliseconds)
+                let event_created_at = Some(
+                    DateTime::from_timestamp_millis((event_timestamp.as_u64() * 1000) as i64)
+                        .ok_or_else(|| {
+                            WhitenoiseError::Database(
+                                crate::whitenoise::database::DatabaseError::InvalidTimestamp {
+                                    timestamp: (event_timestamp.as_u64() * 1000) as i64,
+                                },
+                            )
+                        })?,
+                );
```

Comment:
Maybe we should make it so the methods in nostr_manager already return the timestamp in milliseconds?